packages/fluent/scss/scrollview/_variables.scss

```diff
@@ -0,0 +1,113 @@
+@use "../core/color-system" as *;
+@use "../button/_variables.scss"as *;
+
+/// Width of the border around the scrollview.
+/// @group scrollview
+$kendo-scrollview-border-width: 0 !default;
+/// Font family of the scrollview.
+/// @group scrollview
+$kendo-scrollview-font-family: var( --kendo-font-family, inherit ) !default;
+/// Font size of the scrollview.
+/// @group scrollview
+$kendo-scrollview-font-size: var( --kendo-font-size, inherit ) !default;
+/// Line height of the scrollview.
+/// @group scrollview
+$kendo-scrollview-line-height: var( --kendo-line-height, normal ) !default;
+
+/// Background color of the scrollview.
+/// @group scrollview
+$kendo-scrollview-bg: transparent !default;
```

Comment:
can we use --component-bg here ? Is there any specific reason why you have set transparent ?

Reply:
There is no specific reason.